web/tests/api-spec.js

```diff
@@ -4,13 +4,15 @@ describe('api service', function () {
         fixture.setBase('tests/api-description');
 
         inject(function ($http) {
-            const MOCK_PROJECT_SETTINGS = {
-                API_ROOT: 'localhost:8000'
+            const mockProjectConfig = {
+                getSettings:() => ({
+                    apiRoot: 'localhost:8000'
+                })
```

Comment:
Well, I can see it less readable for someone who only did program `C` or `Java` before, but python doesn't have `function` keyword at all for example, and in `.NET` or `coffescript`, or `ruby` you do have arrows so for devs coming from these background it would be readable.

We're using angular which for anyone who has no experience in it is a tough thing to understand. So the argument that using arrows functions, a standard ES6 feature which will be all over the place makes things more difficult is not so valid imo.